packages/sanity/src/core/components/documentStatus/DocumentStatus.tsx

```diff
@@ -67,6 +72,18 @@ export function DocumentStatus({
     ? versionDateAbsolute || draftDateAbsolute
     : versionUpdatedTimeAgo || draftUpdatedTimeAgo
 
+  const {title} = currentGlobalBundle || {}
+
+  const documentStatus = useMemo(() => {
+    if (published && '_id' in published) {
+      return 'published'
+    } else if (version && '_id' in version) {
+      return 'version'
+    }
```

Comment:
Unsure if this is the best way to handle this, however, from what I understand we:
- want to find out if it's a published / version / draft. The reason why I do it here and not further down it's two fold:
  - from what I understand from this earlier version of the prototype we will eventually want to display different things / tone despending on what status the document is in. 
  - the other components should be kept more agnostic to this information (UI)

Reply:
I feel having it here feels good - calculating document status in `DocumentStatus` aligns well